game.py

```diff
@@ -12,6 +12,7 @@ def __init__(self):
         self.deck = None
         self.player_turn = 0
         self.number_of_players = None
+        self.direction_of_play = 1
```

Comment:
Really nice and it works.
The values of the direction of play are 1 or -1. which are not describe a direction value, they are integers. 
suggestion 1: Convert the value to an enum. 
suggestion 2: convert the value to boolean and pick up better naming.